### DISCUSSION TITLE
Happy model EP1 dual TCXO diversity

### DISCUSSION THREAD

**Discussion by Mick51** [ExpressLRS/ExpressLRS#2544] [General]
Hello everyone, I am happy to use a TX elrs gemini, I took Rx Happy model EP1 dual TCXO diversity.  I would have liked to know if they were compatible with Gemini.  I can't find this answer anywhere. 
Thanks in advance.  😉

**Comment by pkendall64** [comment]
Yes, they are compatible.
All Gemini TX modules and all dual (true diversity) receivers are compatible if they are on the same frequency band.

**Reply by Mick51** [reply]
Thank you very much, I had a doubt.  I watched a tutorial from Oscar Liang, because I had activated gemini mode in the tx part but not rx.  It will work better I think....
